zavod/zavod/exporters/delta.py

```diff
@@ -33,6 +37,15 @@ def generate(self) -> Generator[Any, None, None]:
     def finish(self) -> None:
         with open(self.path, "wb") as fh:
             for op in self.generate():
+                if op["op"] in self.counts:
+                    self.counts[op["op"]] += 1
                 write_json(op, fh)
         self.delta.close()
+        self.context.log.info(
+            "Delta export complete",
```

Comment:
Should we add something to this log that makes it easy to filter for? metric="export_ops" ?